modules/transport/http/src/org/apache/axis2/transport/http/HTTPSender.java

```diff
@@ -310,7 +310,6 @@ private void handleResponse(MessageContext msgContext,
         int statusCode = method.getStatusCode();
         HTTPStatusCodeFamily family = getHTTPStatusCodeFamily(statusCode);
```

Comment:
do we need this anymore ? and the method getHTTPStatusCodeFamily ?

Reply:
We do not need this line and the method after the fix. Removed them both in commit cdb42256f6841648687e25969304e19eb24e7591